File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.py
class Solution:
    def mirrorDistance(self, n: int) -> int:
        if n < 10:
            return 0
        rev = 0
        temp = n
        while n > 0:
            d = n % 10
            rev = rev * 10 + d
            n = n // 10
        return abs(temp - rev)
        

File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.cs
public class Solution {
    public int MirrorDistance(int n) {
        if (n < 10)
        return 0;

        int temp = n;
        int rev = 0;

        while(temp > 0){
            int d = temp % 10;
            rev = rev * 10 + d;
            temp = temp / 10;
        }
        
        return Math.Abs(n - rev);
    }
}